docs/source/docsIndex.js

```diff
@@ -0,0 +1,95 @@
+
+'use strict';
```

Comment:
Why not just name this file `docs/source/index.js`?

Reply:
i had renamed it originally for the move to `/scripts` and didnt rename it back, should i rename it back?